Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -13186,9 +13186,15 @@ - (void)systemDidChangeSelectionForInteraction:(BETextInteraction *)interaction
 
 #if ENABLE(WRITING_TOOLS)
 
+// FIXME: (rdar://130540028) Remove uses of the old WritingToolsAllowedInputOptions API in favor of the new WritingToolsResultOptions API, and remove staging.
 - (UIWritingToolsAllowedInputOptions)writingToolsAllowedInputOptions
 {
-    return [_webView writingToolsAllowedInputOptions];
+    return [_webView allowedWritingToolsResultOptions];
+}
+
+- (UIWritingToolsAllowedInputOptions)allowedWritingToolsResultOptions
```

Comment:
I think this might end up being a build failure once UIKit updates.

Reply:
fixed, and verified it builds fine across all configurations